modules/standard/VisualDebug.chpl

```diff
@@ -31,6 +31,10 @@ module VisualDebug
 
   use String;
 
+  // VDebugOn -- Should we generate data
+  config param DefaultVDebugOn = true;
+  config var VDebugOn = DefaultVDebugOn;
```

Comment:
This was chosen to match the methods...  startVdebug(), stopVdebug() .... and I just noticed the  difference in capitalization.   Also, I thought DefaultVisualDebugOn was a little long.   I had considered Enabled instead of On, but I thought that was even longer ...   I can make it longer.